_data/topnav.yml

```diff
@@ -1,15 +1,21 @@
 ## Topnav single links
 ## if you want to list an external url, use external_url instead of url. the theme will apply a different link base.
-topnav:
-- title: Topnav
-  items:
-    - title: API Docs
-      external_url: http://apidocs.loopback.io
+## topnav:
+## - title: Topnav
+##   items:
+##     - title: API Docs
+##       external_url: http://apidocs.loopback.io
```

Comment:
Do we want to remove them? or keep them commented

Reply:
you're right. Let me remove it. Thanks.